system/doc/reference_manual/errors.xml

```diff
@@ -130,7 +130,11 @@
           <tag><c>error_info</c></tag>
           <item>The second element of the tuple is a map
 	  providing additional information about what caused the
-	  exception.
+	  exception. This information can be created by calling
+          <seemfa marker="erts:erlang#error/3"><c>erlang:error/3</c></seemfa>
```

Comment:
```suggestion
<seemfa marker="erts:erlang#error/3"><c>error/3</c></seemfa>
```

`error/3` does not need any `erlang` prefix. Not sure whether the marker part should be `"erts:erlang#error/3"` or "erts:#error/3".